graphql/resolvers.go

```diff
@@ -4084,6 +4084,20 @@ func (r *ticketFieldsResolver) ResolutionName(ctx context.Context, obj *thirdpar
 
 func (r *Resolver) TicketFields() TicketFieldsResolver { return &ticketFieldsResolver{r} }
 
+func (r *taskResolver) Ami(ctx context.Context, obj *restModel.APITask) (*string, error) {
+	if obj.HostId != nil {
```

Comment:
This will actually always be not nil since we set it regardless of whether or not it's "", so we should check the empty string specifically.